genai-perf/genai_perf/inputs/input_constants.py

```diff
@@ -70,6 +70,7 @@ def to_lowercase(self):
 DEFAULT_SYNTHETIC_FILENAME = "synthetic_data.json"
 DEFAULT_WARMUP_REQUEST_COUNT = 0
 DEFAULT_BACKEND = "tensorrtllm"
+DEFAULT_PROMETHEUS_PORT = 8002
```

Comment:
This is the default port for the Tritonserver metrics. I would change this to be 9090 which is what prometheus uses in its getting started configuration file.

Reply:
Great feedback. Done.